CHANGELOG.md

```diff
@@ -1,3 +1,53 @@
+# v??? (2019-08-20)
+
+⚠️ This release includes breaking changes to `RSpec::Determinator` ⚠️
```

Comment:
I feel like this file follows a convention where features are listed as a one liner under either "Bug fix", "Feature" or "Breaking change". I think it might make sense to have this text as a description for this PR and reference the PR in here.

Reply:
updated